packages/core/src/Stores/contract-trade-store.js

```diff
@@ -238,18 +240,28 @@ export default class ContractTradeStore extends BaseStore {
                 const trade_type_is_supported = trade_types.indexOf(info.contract_type) !== -1;
                 // both high_low & rise_fall have the same contract_types in POC response
                 // entry_spot=barrier means it is rise_fall contract (blame the api)
-                if (trade_type_is_supported && info.barrier && info.entry_tick && is_call_put) {
-                    if (`${+info.entry_tick}` === `${+info.barrier}`) {
-                        return trade_type === 'rise_fall' || trade_type === 'rise_fall_equal';
-                    }
-                    return trade_type === 'high_low';
+                // if (trade_type_is_supported && info.barrier && info.entry_tick && is_call_put) {
+                //     if (`${+info.entry_tick}` === `${+info.barrier}`) {
+                //         return trade_type === 'rise_fall' || trade_type === 'rise_fall_equal';
+                //     }
+                //     return trade_type === 'high_low';
+                // }
+                // 2
+                // console.log('test', info);
```

Comment:
are we ready for code review? clickup card status is in code review

Reply:
@henry-deriv I added a thread message that it wasn't ready. Was not able to update card status in CU as it was laggy

Reply:
<img width="1522" alt="Screenshot 2023-08-28 at 11 39 54 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/400544c1-fc69-4d9e-b7f4-2b9ec65d69d1">